tap_pipedrive/stream.py

```diff
@@ -70,8 +70,7 @@ def has_data(self):
 
     def paginate(self, response):
         payload = response.json()
-
-        if 'additional_data' in payload and 'pagination' in payload['additional_data']:
+        if payload.get('additional_data') and 'pagination' in payload['additional_data']:
```

Comment:
```suggestion
        if 'pagination' in payload.get('additional_data', {}):
```

Reply:
`'pagination' in payload.get('additional_data', {})` fails  for `payload['additional_data']=None`  fails so reverting back to earlier condition.